src/app/components/tasks/task-list/task-list.component.css

```diff
@@ -12,3 +12,7 @@ app-task-item{
   display: flex;
   flex-wrap: wrap;
 }
+
+/deep/ .highlated > mat-card{
```

Comment:
typo? ;) also, its a good practice not to use `!important` statements, because as the app scales up it will at some point become impossible to maintain the CSS :)